nbconvert/_version_tools.py

```diff
@@ -0,0 +1,83 @@
+# import pep440 # cannot be imported as described below
+import re 
```

Comment:
Yeah, put an explicit Github link and a commit ID, for good measure.

I don't personally mind about the copyright notice & license text, and @Carreau may well not mind either. In general, though, if we're including code from another project, we should include those things - it's a typical condition of the licenses which allow you to redistribute that code.